lesson_25/dasiaenglish/src/pages/Home/Program.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+const Program: React.FC= () => {
```

Comment:
Misses the technical requirement:

| The `Program` component should allow you to configure the title and description to display.